javascript-practice/src/pages/dashboard.html

```diff
@@ -27,23 +31,27 @@ <h2 id="fullName" class="header-dashboard-profile-name"></h2>
               </div>
               <p id="role" class="header-dashboard-profile-sub-name"></p>
             </div>
-            <div class="flex-column">
-              <img
-                id="arrow"
-                class="arrow"
-                src="../assets/icons/arrow.svg"
-                alt="arrow"
-              />
-              <span
-                id="logout"
-                class="header-dashboard-profile-sub-name logout"
-              >
-                Logout
-              </span>
-            </div>
+          </div>
+          <div class="flex-column">
+            <img
+              id="arrow"
+              class="arrow"
+              src="../assets/icons/arrow.svg"
+              alt="arrow"
+            />
+            <span id="logout" class="header-dashboard-profile-sub-name logout">
+              Logout
+            </span>
           </div>
         </div>
       </header>
+      <aside class="slide-bar">
+        <img src="../assets/icons/logo.svg" alt="logo" />
+        <div class="flex algin-center dashboard-box">
```

Comment:
Please check design, this's not only for dashboard-box. Btw, I don't think we should use div here